packages/components/src/navigator/navigator-screen/component.tsx

```diff
@@ -151,62 +136,16 @@ function UnconnectedNavigatorScreen(
 		return null;
 	}
 
-	if ( prefersReducedMotion ) {
-		return (
+	return (
+		<>
```

Comment:
There's no need for `Fragment` wrappers since we only return the `View` component and no siblings.

Reply:
Good catch, used that while testing to see where the animations came from